blockchain-agent-proof/src/main/java/com/sphereon/alfresco/blockchain/agent/proof/config/TokenUpdater.java

```diff
@@ -1,48 +1,42 @@
-package com.sphereon.alfresco.blockchain.agent.config;
+package com.sphereon.alfresco.blockchain.agent.proof.config;
```

Comment:
Givven the packagename, suggest to update the className to AuthTokenUpdater as it is not immediately clear this is about authentication

Reply:
Done